### DISCUSSION TITLE
how can I type this symbol with typst

### DISCUSSION THREAD

**Discussion by supikachu** [typst/typst#3817] [General]
### Description

I can use \overparen{AB} and \underparen{ABC} using latex, but I don't know how to type the same symbol with typst.
![11111.png](https://img2.imgtp.com/2024/03/29/6G5ikNuK.png)

### Operating system

Windows

### Typst version

- [X] I am using the latest version of Typst

**Comment by frozolotl** [comment]
You can use `accent("AB", paren.t)` and `accent("ABC", paren.b)` for that.

#3564 will make that a bit easier.
If you have an opinion on extra aliases like `overparen` or `overarc`, please write about them there.

**Comment by Enivex** [comment]
Please don't use issues to ask questions. Instead use the Discussions tab, or the Discord.
